src/libraries/System.IO.FileSystem/tests/Directory/CreateDirectory.cs

```diff
@@ -277,8 +299,9 @@ public void ExtendedDirectoryLongerThanLegacyMaxPath_Succeeds()
         }
 
         [ConditionalFact(nameof(AreAllLongPathsAvailable))]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, ".NET Framework cannot handle long paths")]
```

Comment:
Framework should work with this if it is properly enabled for long paths. These tests should get the `        [ConditionalFact(nameof(AreAllLongPathsAvailable))]` and [other conditions that are exposed in the base test class](https://github.com/dotnet/runtime/blob/4f9ae42d861fcb4be2fcd5d3d55d5f227d30e723/src/libraries/System.IO.FileSystem/tests/FileSystemTest.cs#L18-L22).

Reply:
Ah thanks, good to know. I fixed the .NET Framework runs first, which is why I added the `SkipOnTargetFramework` first, then I  added the `ConditionalFact` second, after realizing .NET Core could break with long paths if they were not supported.